controller/pkg/slackbot/slackbot.go

```diff
@@ -21,9 +22,9 @@ type Slack struct {
 
 func NewSlack(token string, g github.EmojiLookup) *Slack {
 	api := slack.New(token)
-	logger := log.New(os.Stdout, "slack-bot: ", log.Lshortfile|log.LstdFlags)
-	slack.SetLogger(logger)
-	api.SetDebug(false)
+	//logger := log.New(os.Stdout, "slack-bot: ", log.Lshortfile|log.LstdFlags)
```

Comment:
do we need that logging? if not just remove the lines instead of commenting them out